mono/metadata/object.c

```diff
@@ -1132,13 +1132,14 @@ add_imt_builder_entry (MonoImtBuilderEntry **imt_builder, MonoMethod *method, gu
 static void
 print_imt_entry (const char* message, MonoImtBuilderEntry *e, int num) {
 	if (e != NULL) {
+		MonoMethod *method = e->key;
 		printf ("  * %s [%d]: (%p) '%s.%s.%s'\n",
 				message,
 				num,
-				e->method,
-				e->method->klass->name_space,
-				e->method->klass->name,
-				e->method->name);
+				method,
```

Comment:
Yeah unrelated. When I was attempting to debug the IMT stuff this function did not work so I fixed it. Just pushing the fix back so other people don't have to. :-)